docs/DataMutations.md

```diff
@@ -82,4 +82,21 @@ creator.dispatch();
 The `dispatch()` call returns a `Parse.Promise`, so that you can respond when
 server request succeeds or fails. In case of failure, you may want to update
 the state of your application to display a message to the user, and possibly
-give them a click-to-retry option.
\ No newline at end of file
+give them a click-to-retry option.
+
+Note: the above will send three individual requests to the server. Often it is
+desirable to group multiple mutation call into one batch request:
+
+```js
+var batch = new ParseReact.Mutation.Batch();
+creator.dispatch({batch: batch});
```

Comment:
nit: we tend to use whitespace padding when object primitives are a single line
`{batch: batch}` becomes `{ batch: batch }`